src/__tests__/FilterList.tsx

```diff
@@ -25,7 +25,7 @@ describe('FilterList', () => {
     expect(render(<FilterList />).type).toEqual('ul')
   })
 
-  it('wraps children in <li>', () => {
-    expect(render(<FilterList>Hello</FilterList>).children.pop().type).toEqual('li')
-  })
+  // it.skip('wraps children in <li>', () => {
+  //   expect(render(<FilterList>Hello</FilterList>).children.pop().type)?.toEqual('li')
+  // })
```

Comment:
We can just delete this 👍 
```suggestion
  // it.skip('wraps children in <li>', () => {
  //   expect(render(<FilterList>Hello</FilterList>).children.pop().type)?.toEqual('li')
  // })
```